jekyll/_cci2/deploy-to-heroku.adoc

```diff
@@ -23,21 +26,18 @@ link:https://www.heroku.com/[Heroku] is a popular platform for hosting applicati
 3. Use the link:https://circleci.com/developer/orbs/orb/circleci/heroku[Heroku orb] to keep your config simple. The `deploy-via-git` installs the Heroku CLI in the primary container, runs any pre deployment steps you define, deploys your application, then runs any post-deployment steps you define. See the Heroku orb page in the link:https://circleci.com/developer/orbs/orb/circleci/heroku[orbs registry] for full details of parameters and options:
 +
 ```yaml
-version: 2.1
-
+version: '2.1'
 orbs:
-    heroku: circleci/heroku@x.y # Use the Heroku orb in your config
-
```

Comment:
Can we add the replace version warning here too? I guess we will need an adoc version of the snippet 😭

Reply:
oh yes thank you I actually meant to do something about this and forgot 👍